test/bigtest/tests/PurchaseOrder/close-order-modal-test.js

```diff
@@ -0,0 +1,66 @@
+import { describe, beforeEach, it } from '@bigtest/mocha';
+import { expect } from 'chai';
+
+import setupApplication from '../../helpers/setup-application';
+import CloseOrderModal from '../../interactors/PurchaseOrder/close-order-modal';
+import OrderDetailsPage from '../../interactors/order-details-page';
+import { WORKFLOW_STATUS } from '../../../../src/components/PurchaseOrder/Summary/FieldWorkflowStatus';
+
+const REASON = 'test reason';
+
+describe('Close Order Modal', () => {
+  setupApplication();
+
+  let order = null;
+  const modal = new CloseOrderModal();
+  const page = new OrderDetailsPage();
+
+  beforeEach(async function () {
```

Comment:
I think `async` could be omitted, as well as `await`.

Reply:
fixed